### ISSUE TITLE
Feature request:  remove pop-up user profile boxes

### ISSUE THREAD

**Issue by mskala** [oslego/better-twitter#2]
I'm not sure if this is within scope for your project, but it's the one thing I wish I could have from a Twitter-related Firefox extension, and better-twitter looks like a project that might take it on.

When mousing over a user's name in the Twitter Web interface, Twitter pops up a large tooltip-style box showing the user's avatar, profile description, a follow button, etc.  I want that to not happen at all.  The boxes are annoying and serve very little purpose - when I want to see that information I will click through to the user's profile.  So far I haven't figured out a way to use my ad blocker to block the boxes because they are apparently generated on the fly by Javascript, with obfuscated and randomly changing CSS class names.  How about a better-twitter option to filter them out?

**Comment by oslego** [comment]
Sounds like a reasonable request and in spirit with the distraction-free ethos of Better Twitter.
Added an option to hide user profile tooltips (default off) in version 1.3

Mozilla Add-ons: https://addons.mozilla.org/en-US/firefox/addon/better-twitter-extension/
Chrome WebStore: https://chrome.google.com/webstore/detail/better-twitter/illmpnnkeobcgnnjghammeohfjpjoljp

As you have noticed, the new Twitter markup is obfuscated and it's difficult to instrument from an extension. This new feature is not very resilient, but it should work so long as the Twitter markup structure doesn't change drastically. Do file a new issue if it stops working.

**Comment by mskala** [comment]
Nope... unfortunately, even with a copy I checked out with Git and edited to turn the option on by default (because I couldn't change the settings within Firefox), it doesn't work.  The cards still pop up.

**Comment by oslego** [comment]
@mskala Were you able to get the extension working again?

I just tried it twice right now on Chrome and Firefox from their respective stores and it worked without fault. Unfortunately, I'm unable to replicate the issues you're describing with the empty options panel.

**Comment by mskala** [comment]
No, I have never seen an indication that it was working at all.  ("Again" isn't an issue; I had never installed it until this feature was added.)  Other features of this extension don't seem to be taking effect either, so I think my problem is with the extension as a whole rather than with the popup-hiding in particular.  I tried disabling all my other extensions, thinking maybe something like NoScript or ad blocking might be somehow blocking this extension too, but that didn't make a visible difference.
